src/Workspaces/Core/Portable/Shared/Extensions/INamedTypeSymbolExtensions.cs

```diff
@@ -397,6 +397,7 @@ private static bool IsAbstractPropertyImplemented(INamedTypeSymbol classOrStruct
                     if (!propertySymbol.IsReadOnly &&
                         !propertySymbol.IsWriteOnly &&
                         !propertySymbol.IsStatic &&
+                        !propertySymbol.IsAbstract &&
```

Comment:
I don't think this will solve the issue in my first example in https://github.com/dotnet/roslyn/issues/23664#issuecomment-388163437

Also I think we need a more general solution, as the problem is not only related to abstract/concrete properties.